src/Compilers/Core/Portable/DiagnosticAnalyzer/AnalyzerManager.cs

```diff
@@ -436,10 +443,16 @@ internal static bool HasCompilerOrNotConfigurableTag(ImmutableArray<string> cust
         }
 
         internal static bool HasNotConfigurableTag(ImmutableArray<string> customTags)
+            => HasCustomTag(customTags, WellKnownDiagnosticTags.NotConfigurable);
+
+        internal static bool HasCustomSeverityConfigurableTag(ImmutableArray<string> customTags)
+            => HasCustomTag(customTags, WellKnownDiagnosticTags.CustomSeverityConfigurable);
+
+        private static bool HasCustomTag(ImmutableArray<string> customTags, string tagToFind)
         {
             foreach (var customTag in customTags)
```

Comment:
>foreach (var customTag in customTags) [](http://example.com/codeflow?start=12&length=37)

Not part of this change, but it looks like the loop can be replaced with `return customTags.Contains(tagToFind);`.